setup.py

```diff
@@ -0,0 +1,29 @@
+import setuptools
+import versioneer
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
+setuptools.setup(
+    name="versioned-hdf5",
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    author="Quansight",
+    description="Versioned HDF5",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Quansight/versioned-hdf5",
+    packages=['versioned_hdf5', 'versioned_hdf5.tests'],
+    license="MIT",
```

Comment:
I don't know why the license file says MIT when it is clearly BSD-3-Clause (which is what we should use anyway)

Reply:
The license is MIT. I also included the license from h5py because I copied some code from there. I can switch it to BSD if that is better.

Reply:
When I read the text of the license it was BSD, it just said MIT at the top. In any event, yeah, let's switch to BSD because it is a better license